account_analytic_project/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================================
+Project and analytic account integration
+========================================
+
+Gives the option to create projects directly from the analytic accounts they
+should be linked to.
+
+Without this module, if you first create an analytic account and later on you
+decide to have a project as well for this account, the situation is very
+difficult to rectify.
+
```

Comment:
As the solution is one sentence explanation far, I still don't see the problem on teaching the user, but I won't block the PR.